qa/vagrant/build.gradle

```diff
@@ -64,7 +64,8 @@ String upgradeFromVersion
 
 String maybeTestsSeed = System.getProperty("tests.seed", null);
 if (maybeTestsSeed != null) {
-  seed = new BigInteger(maybeTestsSeed, 16).longValue()
+  def (masterSeed, testSeed) = maybeTestsSeed.tokenize(':')
+  seed = new BigInteger(masterSeed, masterSeed.size()).longValue()
```

Comment:
> I'm not sure, this will try to parse abc:def as a ternary value

Did not fail when I tried but I can change that anyway.
> I'm not sure, this will try to parse abc:def as a ternary value and that will fail.

Did not when I tried but I can change.

> Also, shouldn't it be masterSeed.length())?

Can be both I think but I have no preference. Will change.